installer/bundle/bundle_skel_AIX.sh

```diff
@@ -0,0 +1,432 @@
+#!/bin/sh
+
+#
+# Shell Bundle installer package for the SCX project
+#
+
+PATH=/usr/bin:/usr/sbin:/bin:/sbin
+umask 022
+
+# Can't use something like 'readlink -e $0' because that doesn't work everywhere
+# And HP doesn't define $PWD in a sudo environment, so we define our own
+case $0 in
+    /*|~*)
+        SCRIPT_INDIRECT="`dirname $0`"
+        ;;
+    *)
+        PWD="`pwd`"
+        SCRIPT_INDIRECT="`dirname $PWD/$0`"
+        ;;
+esac
+
+SCRIPT_DIR="`(cd \"$SCRIPT_INDIRECT\"; pwd -P)`"
+SCRIPT="$SCRIPT_DIR/`basename $0`"
+EXTRACT_DIR="`pwd -P`/scxbundle.$$"
+DPKG_CONF_QUALS="--force-confold --force-confdef"
+
+# These symbols will get replaced during the bundle creation process.
+#
+# The OM_PKG symbol should contain something like:
+#       scx-1.5.1-115.rhel.6.x64 (script adds .rpm or .deb, as appropriate)
+# Note that for non-Linux platforms, this symbol should contain full filename.
+#
+# PROVIDER_ONLY is normally set to '0'. Set to non-zero if you wish to build a
+# version of SCX that is only the provider (no OMI, no bundled packages). This
+# essentially provides a "scx-cimprov" type package if just the provider alone
+# must be included as part of some other package.
+
+TAR_FILE=<TAR_FILE>
+OM_PKG=<OM_PKG>
+OMI_PKG=<OMI_PKG>
+PROVIDER_ONLY=0
```

Comment:
See above. This can be removed. I imagine this is in all of the bundle files; I won't mention this again for the other bundle files, but please do remove this (and associated 'if' tests everywhere).